doc/manuals/install_guide.rst

```diff
@@ -77,6 +77,13 @@ the last couple of steps, Fletch should build without any issues.
 
 `make`
 
+.. note::
+
+  If you are on Windows, and you wish to use Google Test from Fletch (e.g.
```

Comment:
I've not convinced that this is just an issue with Google Test and unit tests.  I think you always need to match CMAKE_BUILD_TYPE and BUILD_SHARED_LIBS between Fletch and kwiver on Windows.  I think that more than just the unit tests will have problems if you don't.  It was only the unit tests before because of a bug in Fletch where only Google Test did not have a matching BUILD_SHARED_LIBS setting.

Reply:
Okay, I'll remove the testing-specific bits.